src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerConfigurations/AnonymizerConfiguration.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
+using Fhir.Anonymizer.Shared.Core.AnonymizerConfigurations;
```

Comment:
Which class is included in this namespace? It seems all of our classes are in namespace starts with `Microsoft.Fhir...`.

Reply:
Thanks, fixed it.